### ISSUE TITLE
No chamfer distance in the installed tensorflow-graphics-gpu 1.0.0 package

### ISSUE THREAD

**Issue by Enigma-li** [tensorflow/graphics#473]
Hi,

I installed the tensorflow-graphics-gpu as instructed [here](https://tensorflow.google.cn/graphics/install), but this version does not contain the chamfer distance.

How to resolve this?

**Comment by johanesmikhael** [comment]
Hello, I had the same problem too. I am using tf 2.4 python 3.6 in Windows 10 with gpu enabled. I installed tfg but I realized in nn module only layer module installed, metric and loss module are missing. Do I missed something?

**Comment by melhousni** [comment]
Same problem here. Any fix??

**Comment by alimoeeny** [comment]
exact same problem here, any solutions?

**Comment by msokoloff1** [comment]
Use: from tensorflow.graphics.nn.loss import <loss fn>

**Comment by dvbuntu** [comment]
The `loss` module was only added in [early 2020](https://github.com/tensorflow/graphics/blame/master/tensorflow_graphics/nn/loss/chamfer_distance.py#L26-L89), so it's not in the prebuilt `pip` version (dated May 2019).  You can try to install the most recent version with [`https://pypi.org/project/tensorflow-graphics/2021.10.21/`](https://pypi.org/project/tensorflow-graphics/2021.10.21/)

**Comment by Eastmanmd** [comment]
Was this problem fully resolved ? I tried installing the most recent version and I am still having the same problem when trying to use chamfer distance.

**Comment by DoubangoTelecom** [comment]
```
from tensorflow_graphics.nn.loss import chamfer_distance
loss = chamfer_distance.evaluate(p1, p2)
```
